.gitignore

```diff
@@ -0,0 +1,3 @@
+# IDE
+*.iml
+.idea
```

Comment:
These belong in your global Git ignores so they do not need to be repeated in every project.